app/views/newbookings/new.html.erb

```diff
@@ -4,6 +4,9 @@
 <body>
 <%= form_for [@customer, @pet, @newbooking] do |form| %>
   <%= form.datetime_select :session %>
+  <p>
+  <%= form.collection_select(:service_id, Service.all, :id, :name, :prompt => 'Select') %>
```

Comment:
Although `:prompt => 'Select'` is correct, `prompt: 'Select'` is the preferred style. It means exactly the same, but is shorter (and more modern). You'll see syntax both all over the place...

Reply:
I see. I'll change that now.